models/repo_issue.go

```diff
@@ -32,3 +32,22 @@ func (repo *Repository) AllowOnlyContributorsToTrackTime() bool {
 	}
 	return u.IssuesConfig().AllowOnlyContributorsToTrackTime
 }
+
+/*
+________                                   .___                   .__
+\______ \   ____ ______   ____   ____    __| _/____   ____   ____ |__| ____   ______
+ |    |  \_/ __ \\____ \_/ __ \ /    \  / __ |/ __ \ /    \_/ ___\|  |/ __ \ /  ___/
+ |    `   \  ___/|  |_> >  ___/|   |  \/ /_/ \  ___/|   |  \  \___|  \  ___/ \___ \
+/_______  /\___  >   __/ \___  >___|  /\____ |\___  >___|  /\___  >__|\___  >____  >
+        \/     \/|__|        \/     \/      \/    \/     \/     \/        \/     \/
+*/
```

Comment:
does it make sense to put thi sin models/repo_issue_dependencies.go ?

Reply:
I think so, it is directly related to issues. Why wouldn't it make sense?

Reply:
I can't think of any reason, so please move it ?

Reply:
Sorry I misunderstood you, moved it.